src/test/java/org/sagebionetworks/bridge/sdk/integration/UTF8Test.java

```diff
@@ -11,57 +10,23 @@
 import org.sagebionetworks.bridge.sdk.TestUserHelper;
 import org.sagebionetworks.bridge.sdk.TestUserHelper.TestUser;
 import org.sagebionetworks.bridge.sdk.UserClient;
-import org.sagebionetworks.bridge.sdk.models.ResourceList;
-import org.sagebionetworks.bridge.sdk.models.holders.GuidVersionHolder;
-import org.sagebionetworks.bridge.sdk.models.studies.Tracker;
-import org.sagebionetworks.bridge.sdk.models.users.HealthDataRecord;
 import org.sagebionetworks.bridge.sdk.models.users.SignInCredentials;
 import org.sagebionetworks.bridge.sdk.models.users.UserProfile;
 
-import com.fasterxml.jackson.databind.node.JsonNodeFactory;
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import com.google.common.collect.Lists;
-
 public class UTF8Test {
     
     private TestUser testUser;
     
     @Before
     public void before() {
-        testUser = TestUserHelper.createAndSignInUser(HealthDataTest.class, true);
+        testUser = TestUserHelper.createAndSignInUser(UTF8Test.class, true);
     }
     
     @After
     public void after() {
         testUser.signOutAndDeleteUser();
     }
-    
-    @Test
-    public void canSaveAndRetrieveUTF8HealthData() {
```

Comment:
I just wanted to verify that UTF-8 is working end-to-end, can we do this test with any other entity, like a survey or a study?